Failed to Build on Heroku

```
remote: Compressing source files... done.
remote: Building source:
remote: 
remote: -----> meteor app detected
remote: 
remote: -----> Moving app source into a subdirectory
remote: 
remote:        Node engine:         0.10.40
remote:        Npm engine:          unspecified
remote:        Start mechanism:     none
remote:        node_modules source: none
remote:        node_modules cached: false
remote: 
remote:        NPM_CONFIG_PRODUCTION=true
remote:        NODE_MODULES_CACHE=true
remote: 
remote:        PRO TIP: Use 'npm init' and 'npm install --save' to define dependencies
remote:        See https://devcenter.heroku.com/articles/nodejs-support
remote: 
remote:        PRO TIP: Include a Procfile, package.json start script, or server.js file to start your app
remote:        See https://devcenter.heroku.com/articles/nodejs-support#runtime-behavior
remote: 
remote: -----> Installing binaries
remote:        Downloading and installing node 0.10.40...
remote: 
remote: -----> Building dependencies
remote:        Skipping dependencies (no source for node_modules)
remote: 
remote: -----> Checking startup method
remote: 
remote: -----> Finalizing build
remote:        Creating runtime environment
remote:        Exporting binary paths
remote:        Cleaning up build artifacts
remote:        Build successful!
remote:        /tmp/build_3cd55c27749d4d8e53feccc640e5392d
remote:        └── (empty)
remote:        
remote: 
remote: -----> Fetching Meteor 1.7.0.3
remote:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
remote:                                  Dload  Upload   Total   Spent    Left  Speed
remote:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--remote:   0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--remote:   0  173M    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--remote:  10  173M   10 18.3M    0     0  7756k      0  0:00:22  0:00:02  0:00:20remote:  36  173M   36 63.9M    0     0  18.5M      0  0:00:09  0:00:03  0:00:06remote:  69  173M   69  119M    0     0  27.0M      0  0:00:06  0:00:04  0:00:02remote: 100  173M  100  173M    0     0  32.7M      0  0:00:05  0:00:05 --:--:-- 42.6M
remote: 
remote: -----> Unpacking Meteor 1.7.0.3
remote:        Meteor 1.7.0.3 is installed
remote: 
remote: -----> Building Meteor App Bundle
remote:        WARNING: npm peer requirements (for juliancwirko:postcss) not installed:
remote:         - postcss@^6.0.22 not installed.
remote:         - postcss-load-config@^1.2.0 not installed.
remote:        
remote:        Read more about installing npm peer dependencies:
remote:          http://guide.meteor.com/using-packages.html#peer-npm-dependencies
remote:        
remote:        Errors prevented bundling:
remote:        While loading plugin `minifier-postcss` from package `juliancwirko:postcss`:
remote:        packages/modules-runtime.js:231:12: Cannot find module 'postcss'
remote:        at makeMissingError (packages/modules-runtime.js:231:12)
remote:        at require (packages/modules-runtime.js:241:19)
remote:        at minify-css.js (packages/minifier-postcss/plugin/minify-css.js:16:15)
remote:        at fileEvaluate (packages/modules-runtime.js:343:9)
remote:        at require (packages/modules-runtime.js:238:16)
remote:        at <runJavaScript-23>:279:1
remote:        at <runJavaScript-23>:284:3
remote:        
remote:        
remote: 
remote: -----> Installing App's NPM Dependencies
remote: /app/tmp/buildpacks/48bbcd8a1f2d1885ea7beef562aaaa72c82d8575e00fec23e8342bcc96a462c83fe9b01b1c05c977c72a3e06a800ba84ace0a00796dcb33135260c214066cb33/bin/compile_meteor: line 66: cd: ../build/bundle/programs/server: No such file or directory
remote:  !     Push rejected, failed to compile meteor app.
remote: 
remote:  !     Push failed
remote: Verifying deploy...
remote: 
remote: !	Push rejected to jimmyxue.
remote: 
To https://git.heroku.com/jimmyxue.git
 ! [remote rejected] master -> master (pre-receive hook declined)
error: failed to push some refs to 'https://git.heroku.com/jimmyxue.git'

```

--- a/README.md
+++ b/README.md
@@ -24,7 +24,7 @@ Verify the installation.
     meteor --version
 ```
 
-### Deployment
+### Build
 
 Fork or clone this repository.
 
@@ -34,6 +34,14 @@ Fork or clone this repository.
     meteor npm install
     meteor
 ```
+
+### Deployment
+
+You may use this quick setup to deploy on [Heroku](https://dashboard.heroku.com/).
+
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/pkcwong/meteor-react-starter/tree/master)
+
+Demo App: https://meteor-react-starter.herokuapp.com/
 
 ### Support and Update
 

--- a/app.json
+++ b/app.json
@@ -10,10 +10,6 @@
     "ROOT_URL": {
       "description": "Modify accordingly https://<PROJECT_NAME>.herokuapp.com",
       "value": "https://<PROJECT_NAME>.herokuapp.com"
-    },
-    "MONGO_URL": {
-      "description": "Copy from <MONGODB_URI> after deployment",
-      "value": "<MONGODB_URI>"
     }
   },
   "addons": [

--- a/app.json
+++ b/app.json
@@ -6,14 +6,6 @@
     "meteor"
   ],
   "repository": "https://github.com/pkcwong/meteor-react-starter",
-  "env": {
-    "ROOT_URL": {
-      "description": "https://<PROJECT_NAME>.herokuapp.com"
-    },
-    "MONGO_URL": {
-      "description": "<MONGODB_URI_VALUE>"
-    }
-  },
   "addons": [
     {
       "plan": "mongolab:sandbox"

--- a/app.json
+++ b/app.json
@@ -1,0 +1,27 @@
+{
+  "name": "meteor-react-starter",
+  "description": "A starter repo to building a web application with ReactJS, using the MeteorJS framework.",
+  "keywords": [
+    "react",
+    "meteor"
+  ],
+  "repository": "https://github.com/pkcwong/meteor-react-starter",
+  "env": {
+    "ROOT_URL": {
+      "description": "https://<PROJECT_NAME>.herokuapp.com"
+    },
+    "MONGO_URL": {
+      "description": "<MONGODB_URI_VALUE>"
+    }
+  },
+  "addons": [
+    {
+      "plan": "mongolab:sandbox"
+    }
+  ],
+  "buildpacks": [
+    {
+      "url": "https://github.com/AdmitHub/meteor-buildpack-horse.git"
+    }
+  ]
+}

--- a/app.json
+++ b/app.json
@@ -8,12 +8,12 @@
   "repository": "https://github.com/pkcwong/meteor-react-starter",
   "env": {
     "ROOT_URL": {
-      "description": "https://<PROJECT_NAME>.herokuapp.com",
-      "value": null
+      "description": "Modify accordingly https://<PROJECT_NAME>.herokuapp.com",
+      "value": "https://<PROJECT_NAME>.herokuapp.com"
     },
     "MONGO_URL": {
-      "description": "<MONGODB_URI>",
-      "value": null
+      "description": "Copy from <MONGODB_URI> after deployment",
+      "value": "<MONGODB_URI>"
     }
   },
   "addons": [

--- a/app.json
+++ b/app.json
@@ -6,6 +6,16 @@
     "meteor"
   ],
   "repository": "https://github.com/pkcwong/meteor-react-starter",
+  "env": {
+    "ROOT_URL": {
+      "description": "https://<PROJECT_NAME>.herokuapp.com",
+      "value": null
+    },
+    "MONGO_URL": {
+      "description": "<MONGODB_URI>",
+      "value": null
+    }
+  },
   "addons": [
     {
       "plan": "mongolab:sandbox"

--- a/package.json
+++ b/package.json
@@ -10,7 +10,6 @@
     "bootstrap": "^4.1.1",
     "gridfs-stream": "^1.1.1",
     "meteor-node-stubs": "^0.3.2",
-    "postcss-load-config": "^1.2.0",
     "prop-types": "^15.6.2",
     "react": "^16.4.0",
     "react-bootstrap": "^0.32.1",
@@ -29,6 +28,8 @@
     "babel-core": "^7.0.0-bridge.0",
     "babel-jest": "^23.4.0",
     "jest": "^23.4.1",
+    "postcss": "^6.0.22",
+    "postcss-load-config": "^1.2.0",
     "regenerator-runtime": "^0.12.0"
   },
   "postcss": {

--- a/server/main.js
+++ b/server/main.js
@@ -1,7 +1,6 @@
 import { Meteor } from 'meteor/meteor';
 
 Meteor.startup(() => {
-
 	let root = Meteor.users.findOne({
 		username: 'root'
 	});
@@ -15,5 +14,4 @@ Meteor.startup(() => {
 		});
 		Roles.addUsersToRoles(user, ['root', 'administrator']);
 	}
-
 });

